inc/optimus_management.class.php

```diff
@@ -97,6 +92,20 @@ public static function bulk_optimizer_assets() {
             $id_query = "";
         }
 
+        return array($queryImageTypes, $optimus_query, $id_query);
+    }
+
+    /**
+    * Bulk optimizer collect assets
```

Comment:
Here and in PHPDoc blocks below: Wrong indentation of PHPDoc lines after the first.